runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -32,9 +32,16 @@
 
 extern "C" {
 #include "xaiengine.h"
+#include "xaiengine/xaie_device_aieml.h"
+#define s8
+#define u8
+#define u16
+#define s32
+#define u32
+#define u64
```

Comment:
Are these needed?

Reply:
Yes - otherwise we leak the same defines from `aie-rt` which I would prefer not pop up as  suggestions in anyone's editors. But I just realized I'm doing this wrong and these should just be `#undef`s.

Reply:
I see, the `#undef` makes sense, thanks.